app/bundles/CampaignBundle/Entity/Event.php

```diff
@@ -246,6 +252,11 @@ public static function loadMetadata(ORM\ClassMetadata $metadata)
             ->columnName('channel_id')
             ->nullable()
             ->build();
+
+        $builder->createManyToOne('scoringCategory', 'Mautic\ScoringBundle\Entity\ScoringCategory')
```

Comment:
These changes are no longer needed as per my comment in EventType.php.

Reply:
Maybe we should arrange a meeting with our lead dev @captivea-qch on the topic because we don't see how to achieve the same functionnalities without having this kind of relation between the Event and our ScoringCategory ?